build/yamls/antrea-eks-node-init.yml

```diff
@@ -0,0 +1,72 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  labels:
+    app: antrea
+    component: antrea-node-init
+  name: antrea-node-init
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-node-init
+  template:
+    metadata:
+      labels:
+        app: antrea
+        component: antrea-node-init
+    spec:
+      hostPID: true
+      hostNetwork: true
+      containers:
+        - name: node-init
+          image: gcr.io/google-containers/startup-script:v1
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          env:
+          - name: STARTUP_SCRIPT
+            value: |
+              #! /bin/bash
+
+              set -o errexit
+              set -o pipefail
+              set -o nounset
+
+              if [ -f /opt/cni/antrea-node-init-status ]; then
```

Comment:
should this be under tmpfs mounted directory like /var/run/antrea/ so that it can be reset after restarting? otherwise the race condition could happen again? but not sure if eks has restart case.

Reply:
We dont want to restart in case of reboot. Adding under /var/run/antrea will reset it if worker node is restarted. The idea is if the cni conf file is populated, kubelet will make sure both cnis returns a success before scheduling a pod

Reply:
got it